pyvista/core/filters/data_set.py

```diff
@@ -1474,7 +1474,12 @@ def outline_corners(self, factor=0.2, progress_bar=False):
         _update_alg(alg, progress_bar, 'Producing an Outline of the Corners')
         return wrap(alg.GetOutputDataObject(0))
 
-    def extract_geometry(self, extent: Optional[Sequence[float]] = None, progress_bar=False):
+    def extract_geometry(
+        self,
+        extent: Optional[Sequence[float]] = None,
+        point_dtype='float32',
```

Comment:
Unfortunately this is a breaking change because you have changed the order of the potentail positional args.  In hindsight, it would have been better to make `progress_bar` a keyword only argument everywhere.

Reply:
Thanks! I will fix it to maintain backward compatibility. Let's discuss in the issue whether to make the destructive change to make `progress_bar` a keyword argument.